cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -167,31 +178,71 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
-        "the database will be read-only and prevent write queries.")
-      query("ALTER DATABASE customers SET ACCESS READ ONLY", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 1)
-      })) {
-        statsOnlyResultTable()
+      initQueries("CREATE DATABASE customers")
+      p("Databases can be modified using the command `ALTER DATABASE`. ")
+      section("Access", "administration-databases-alter-database-access") {
+        p("By default, a database has read-write access mode on creation. " +
+          "The database can be limited to read-only mode on creation using the configuration parameters " +
+          "dbms.databases.default_to_read_only, dbms.databases.read_only, and dbms.database.writable. " +
+          "For details, see <<operations-manual#manage-databases-parameters, Configuration parameters>>.")
+        p("A database which was created with read-write access mode can be changed to read-only. " +
+          "To change it to read-only, you can use the `ALTER DATABASE` command with the sub-clause `SET ACCESS READ ONLY`. " +
+          "Subsequently, the database access mode can be switched back to read-write using the sub-clause `SET ACCESS READ WRITE`. " +
+          "Altering the database access mode is allowed at all times, whether a database is online or offline. ")
+        p("If conflicting modes are set by the `ALTER DATABASE` command and the configuration parameters, i.e. one says read-write and the other read-only, " +
+          "the database will be read-only and prevent write queries.")
+        query("ALTER DATABASE customers SET ACCESS READ ONLY", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        p("The database access mode can be seen in the `access` output column of the command `SHOW DATABASES`.")
+        query("SHOW DATABASES yield name, access", assertDatabasesShown) {
+          resultTable()
+        }
+        p("`ALTER DATABASE` commands are optionally idempotent, with the default behavior to fail with an error if the database does not exist. " +
+          "Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the database not exist.")
+        query("ALTER DATABASE nonExisting IF EXISTS SET ACCESS READ WRITE", ResultAssertions(r => {
+          assertStats(r)
+        })) {}
       }
-      p("The database access mode can be seen in the `access` output column of the command `SHOW DATABASES`.")
-      query("SHOW DATABASES yield name, access", assertDatabasesShown) {
-        resultTable()
+      section("Topology", "administration-databases-alter-database-topology") {
+        initQueries("CREATE DATABASE `topology-example`")
+        p("In a cluster environment, it may be desirable to change the number of servers used to host a database. " +
+          "The number of primary and secondary servers can be specified using the following command. " +
```

Comment:
Same as before:

It's better to add the command right after the end of this sentence and move "for more details..." after the query section. Also, please add : after "following command", so it is more demonstrative.